### ISSUE TITLE
Having trouble running this

### ISSUE THREAD

**Issue by cryptoquick** [olix0r/cargo-action-fmt#115]
```
Run olix0r/cargo-action-fmt@release/v1.0.0
Run curl --proto '=https' --tlsv1.3 -fsSLo /usr/local/bin/cargo-action-fmt "https://github.com/diba-io/bitmask-core/releases/download/release%2Fv1.0.0/cargo-action-fmt-x8[6](https://github.com/diba-io/bitmask-core/actions/runs/3238182422/jobs/5306111446#step:4:7)_64-unknown-linux-gnu"
  curl --proto '=https' --tlsv1.3 -fsSLo /usr/local/bin/cargo-action-fmt "https://github.com/diba-io/bitmask-core/releases/download/release%2Fv1.0.0/cargo-action-fmt-x86_64-unknown-linux-gnu"
  chmod [7](https://github.com/diba-io/bitmask-core/actions/runs/3238182422/jobs/5306111446#step:4:8)55 /usr/local/bin/cargo-action-fmt
  shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
  env:
    CARGO_TERM_COLOR: always
curl: (22) The requested URL returned error: 404
```

**Comment by olix0r** [comment]
This was fixed in 59ae295e6d5a31c025f24d53e46566f3f644beba.

You should now be able to use `olix0r/cargo-action-fmt/setup@v2`

**Comment by cryptoquick** [comment]
This seems to work great now, thanks!
